QRCoder/QRCodeGenerator.Polynom.cs

```diff
@@ -43,6 +154,134 @@ public override string ToString()
 
                 return sb.ToString();
             }
+
+            /// <inheritdoc/>
+            public void Dispose()
+            {
+                Free(_polyItems);
+                _polyItems = null;
+            }
+
+            /// <summary>
+            /// Ensures that the polynomial has enough capacity to store the specified number of polynomial terms.
+            /// </summary>
+            private void EnsureCapacity(int min)
```

Comment:
This method (right now, w/ the commented out part) doesn't "ensure" the capacity, rather it
```suggestion
            private void AssertCapacity(int min)
```
😉.

Reply:
Done